mne_bids/dig.py

```diff
@@ -543,6 +546,104 @@ def _read_dig_bids(electrodes_fpath, coordsystem_fpath,
     # put back in unknown for unknown coordinate frame
     if coord_frame == 'unknown':
         for ch in raw.info['chs']:
-            ch['coord_frame'] = _str_to_frame['unknown']
+            ch['coord_frame'] = MNE_STR_TO_FRAME['unknown']
         for d in raw.info['dig']:
-            d['coord_frame'] = _str_to_frame['unknown']
+            d['coord_frame'] = MNE_STR_TO_FRAME['unknown']
+
+
+@verbose
+def template_to_head(raw, space, coord_frame='auto', unit='auto',
+                     verbose=None):
+    """Transform a BIDS standard template montage to the head coordinate frame.
```

Comment:
The template montage is getting transformed to head and you need a head->mri transform to get it to work with functions that require a `raw` and a `trans`

Reply:
Ah yeah I see. Hard to do the mental gymnastics sometimes...

In that case, I wonder if it's worth adding a `Notes` section that explicitly warns the user there is literally no way of checking if you do something wrong (e.g. putting in the incorrect name in `space` parameter). Say someone puts `fsaverage5` instead of `fsaverage6`.

Reply:
Neither `fsaverage5` or `fsaverage6` are supported, those are deprecated haha.

You can plot the alignment with the T1 of the template space, if you put the wrong one, it will look off... We can add that later, there's an issue open because `mne-bids` doesn't support 3D plotting yet so I would have but not this PR